docs/book/currencies.rst

```diff
@@ -36,26 +36,6 @@ To manage the currently used currency, we use the **CurrencyContext**. You can a
         $currency = $this->get('sylius.context.currency')->getCurrency();
     }
 
-Currency Converter
-------------------
-
-The **Sylius\Component\Currency\Converter\CurrencyConverter** is a service available under the ``sylius.currency_converter`` id.
-
-It lets you to convert money values from the base currency to all the other currencies and backwards.
-
-.. code-block:: php
-
-    <?php
-
-    public function fooAction()
-    {
-        // convert 100 of the base currency (for instance 100$ if USD is your base) to PLN.
-        $this->get('sylius.currency_converter')->convertFromBase(100, 'PLN');
-
-        // or the other way - convert 100 PLN to amount in the base currency
-        $this->get('sylius.currency_converter')->convertToBase(100, 'PLN');
-    }
```

Comment:
This service will stay, but the method signature will change. I would make it `convert($fromCurrencyCode, $toCurrencyCode, $amount)`